cmake/dart.pc.in

```diff
@@ -1,5 +1,5 @@
 # This file was generated by CMake for @PROJECT_NAME@
-prefix=@CMAKE_INSTALL_PREFIX@
+prefix=${pcfiledir}/../..
```

Comment:
on Ubuntu, many libraries are installed in a architecture-specific subfolder of `/usr/lib`, like `/usr/lib/x86_64-linux-gnu` and the `.pc` files are relative to that as well, like `/usr/lib/x86_64-linux-gnu/pkgconfig`. So I think `../..` isn't a universal relative path to the prefix.

(it turns out that `dart.pc` isn't actually installed by dart's debian metadata, so I filed https://github.com/dartsim/dart-debian/issues/7, but I think my comment still stands)

Reply:
Probably it is possible to avoid that `../..` being hardcoded by computing the relative_path in cmake between `${CMAKE_INSTALL_PREFIX}/${CMAKE_INSTALL_LIBDIR}` and `${CMAKE_INSTALL_PREFIX}`:
~~~
if(IS_ABSOLUTE ${CMAKE_INSTALL_PREFIX})
  file(RELATIVE_PATH pkg_conf_relative_path ${CMAKE_INSTALL_LIBDIR} ${CMAKE_INSTALL_PREFIX})
else()
  file(RELATIVE_PATH pkg_conf_relative_path ${CMAKE_INSTALL_PREFIX}/${CMAKE_INSTALL_LIBDIR} ${CMAKE_INSTALL_PREFIX})
endif()
~~~
but I am not sure if CMake handles correctly the "inversion" of a path like that. This furthermore will not work on Windows if `${CMAKE_INSTALL_PREFIX}` and `${CMAKE_INSTALL_LIBDIR}` belong to different drives, but I think that that is such a corner case that it does not really matter to cover.

Reply:
Created a new PR inspired by @traversaro's idea: https://github.com/dartsim/dart/pull/1529

I think the corner case on Windows should be fine because `${CMAKE_INSTALL_LIBDIR}` seems to be a relative path (e.g., `lib/` or `lib/x86_64-linux-gnu/`) so the absolute path of the pkgconfig file is always the same drive of `${CMAKE_INSTALL_PREFIX}`.